modules/rostests/apitests/shlwapi/SHPropertyBag.cpp

```diff
@@ -29,6 +30,30 @@ static void ResetTest(VARTYPE vt,
     s_pszPropNames[3] = pszName3;
 }
 
+static SAFEARRAY* CreateByteArray(LPCVOID pvSrc, DWORD cbSize)
+{
+    SAFEARRAYBOUND Bound;
+    Bound.lLbound = 0;
+    Bound.cElements = cbSize;
+
+    SAFEARRAY* pArray = SafeArrayCreate(VT_UI1, 1, &Bound);
+    if (!pArray)
+        return NULL;
+
+    void HUGEP *pvData;
+    HRESULT hr = SafeArrayAccessData(pArray, &pvData);
+    if (FAILED(hr))
+    {
+        SafeArrayDestroy(pArray);
+        return NULL;
+    }
+
+    CopyMemory(pvData, pvSrc, cbSize);
+    SafeArrayUnaccessData(pArray);
+
+    return pArray;
+}
+
```

Comment:
(I cannot add github comments on the code below)
Why using `ok_int(0, 1);` in the `CDummyPropertyBag` functions below? (No comment is present to explain what's going on.)

Reply:
`ok_int(0, 1)` is a just failure.

Reply:
Added comments.

Reply:
OK then please use instead the more understandable syntax:
`ok(FALSE, "An understandable error message that shows in the tests results screen\n");`
(and with a suitable error message)

Reply:
Fixed.